frontend/src/components/Moment.js

```diff
@@ -35,12 +98,51 @@ class Moment extends Component {
                     onClose={() => this.setState({ showDeleteModal: false })}/>
             {this.state.toHome && <Redirect to={'/'} />}
             { (props.currentUser ? props.currentUser.username === props.user.username : false) && <h1 className="delete-button" onClick={this.deleteMoment}>X</h1> }
+            {
+                (props.currentUser ? props.currentUser.username === props.user.username : false) &&
+                !editing &&
+                window.location.pathname === `/moment/${props.momentId}` &&
```

Comment:
Think about passing a props to determine if the moment is editable. This forces hardcoded paths to be added if ever more moments end up being editable